<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransfersWidget</class>
 <widget class="QWidget" name="TransfersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#tvTransfers
{
    background-color: white;
    paint-alternating-row-colors-for-empty-area: false;
}

#sWidget, #wTableHeader
{
    background-color: white;
}

#wPauseResumeVisible
{
    width: 16px;
    min-width: 16px;
    max-width: 16px;
    height: 16px;
    min-height: 16px;
    max-height: 16px;
}

#wTableHeader QPushButton
{
	color: #666666;
	font-size: 12px;
    text-align: left;
	border: none;
	margin-right: 0;
    padding: 0;
}

#lHeaderStatus, #lHeaderTime
{
	color: #666666;
	font-size: 12px;
    text-align: left;
	border: none;
	margin-left: 0;
    padding: 0;
}


#lHeaderSpeed
{
	color: #666666;
	font-size: 12px;
    text-align: left;
	padding-left: 4px;
	margin-left 0;
	border: none;
}

TransferWidgetHeaderItem  QLabel
{
   color: #666666;
   font-size: 12px;
   text-align: left;
   border: none;
   margin-left: 0;
   padding: 0;
}
</string>
  </property>
  <layout class="QGridLayout" name="TransfersWidgetLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="sWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="TransfersStateInfoWidget" name="pNoTransfers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>684</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="pNoTransfersLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="pTransfers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>714</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="pTransfersLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wTableHeader" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="wTableHeaderLayout" stretch="0,0,1,0,0,0,0,0,0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>24</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>48</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="TransferWidgetHeaderItem" name="nameColumn" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="title" stdset="0">
             <string>Name</string>
            </property>
            <property name="sortCriterion" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="TransferWidgetHeaderItem" name="sizeColumn" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>28</height>
             </size>
            </property>
            <property name="title" stdset="0">
             <string>Size</string>
            </property>
            <property name="sortCriterion" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="TransferWidgetHeaderItem" name="speedColumn" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>28</height>
             </size>
            </property>
            <property name="sortCriterion" stdset="0">
             <number>2</number>
            </property>
            <property name="title" stdset="0">
             <string>Speed</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="TransferWidgetHeaderItem" name="statusColumn" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>28</height>
             </size>
            </property>
            <property name="sortCriterion" stdset="0">
             <number>3</number>
            </property>
            <property name="title" stdset="0">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="TransferWidgetHeaderItem" name="timeColumn" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="sortCriterion" stdset="0">
             <number>4</number>
            </property>
            <property name="title" stdset="0">
             <string>Time Left</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QPushButton" name="tPauseResumeVisible">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Pause or resume all selected transfers</string>
            </property>
            <property name="icon">
             <iconset resource="../../../gui/Resources_macx.qrc">
              <normaloff>:/images/transfer_manager/transfers_actions/lists_pause_all_ico_default.png</normaloff>:/images/transfer_manager/transfers_actions/lists_pause_all_ico_default.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRaise" stdset="0">
             <bool>false</bool>
            </property>
            <property name="hover_selected" stdset="0">
             <bool>true</bool>
            </property>
            <property name="icon_prefix" stdset="0">
             <url>
              <string>qrc:/images/transfer_manager/transfers_actions/lists_pause_all_ico_default.png</string>
             </url>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QPushButton" name="tCancelClearVisible">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Cancel all selected transfers</string>
            </property>
            <property name="icon">
             <iconset resource="../../../gui/Resources_macx.qrc">
              <normaloff>:/images/transfer_manager/transfers_actions/lists_cancel_all_ico_default.png</normaloff>:/images/transfer_manager/transfers_actions/lists_cancel_all_ico_default.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="icon_prefix" stdset="0">
             <url>
              <string>qrc:/images/transfer_manager/transfers_actions/lists_cancel_all_ico_default.png</string>
             </url>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="rightMargin">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MegaTransferView" name="tvTransfers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>692</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollBar:vertical
{
    background: #f6f6f6;
    width: 15px;
    border-left: 1px solid #E4E4E4;
}

QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical
{
    border: none;
    background: none;
    width:0px;
    height:0px;
}

QScrollBar::handle:vertical
{
    background: #c0c0c0;
    min-height: 20px;
    border-radius: 4px;
    margin: 3px 4px 3px 2px;
}

QMenu
{
    border: 1px solid #B8B8B8;
   border-radius: 8px;
    background: #ffffff;
    padding-top: 16px;
    padding-bottom: 16px;
    padding-left: 8px;
    padding-right: 8px;
}

QMenu::item
{
  font-size: 14px;
  color: #333333;
background: transparent;
margin-top: 3px;
margin-bottom: 3px;
    padding-left: 16px;
    padding-right: 16px;
    margin-left: 16px;
    margin-right: 16px;
}

QMenu::item:selected
{
     font-weight: bold;  
}

QMenu::separator
{
    height: 1px;
	width: 164px;
	min-width: 164px;
	max-width: 164px;
    background-color: #EEEEEE;
    margin-top: 9px;
    margin-bottom: 9px;
}

QMenu::item:disabled
{
    background-color: #ffffff;
    color: rgba(119,119,119,0.3);
}

QMenu::item:default
{
  color: #F46265;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="autoScrollMargin">
          <number>16</number>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MegaTransferView</class>
   <extends>QTreeView</extends>
   <header>MegaTransferView.h</header>
  </customwidget>
  <customwidget>
   <class>TransfersStateInfoWidget</class>
   <extends>QWidget</extends>
   <header>TransfersStateInfoWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransferWidgetHeaderItem</class>
   <extends>QWidget</extends>
   <header>TransferWidgetHeaderItem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../gui/Resources_macx.qrc"/>
 </resources>
 <connections/>
</ui>
